wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+---
+tags:
+  - UK Community Cup 7
+  - UKCC 7
```

Comment:
likewise no *word* of the title needs to be repeated in the tags if it's not restricting the tag on purpose. here, the "7" is already matched by the title, and I'd want to be able to find this article by searching "UKCC" anyway
```suggestion
  - UKCC
```